src/components/PageWizard/PageWizard.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { ProgressIndicator, ProgressStep } from 'carbon-components-react';
-import classnames from 'classnames';
+import classNames from 'classnames';
```

Comment:
Does this not trigger a lint warning. I think we have a lint rule to standardize these imports to only use `classnames`

Reply:
```suggestion
import classnames from 'classnames';
```